test/athens/parse_helper_test.clj

```diff
@@ -0,0 +1,17 @@
+(ns athens.parse-helper-test
+  (:require
+    [athens.parse-helper :as parse-helper]
+    [clojure.test :refer [deftest is]]))
+
+
+(deftest combine-adjacent-strings-tests
+  (is (= []
+         (parse-helper/combine-adjacent-strings [])))
+  (is (= ["some text"]
+         (parse-helper/combine-adjacent-strings ["some" " " "text"])))
+  (is (= ["some text" [:link] "around a link"]
+         (parse-helper/combine-adjacent-strings ["some" " " "text" [:link] "around " "a link"])))
+  (is (= [{:something nil} "more text" [:link] "between elements" 39]
+         (parse-helper/combine-adjacent-strings [{:something nil} "more" " " "text" [:link] "between" " " "elements" 39])))
+  (is (= [{:a 1 :b 2} 3 ["leave" "intact"]]
+         (parse-helper/combine-adjacent-strings [{:a 1 :b 2} 3 ["leave" "intact"]]))))
```

Comment:
For a lot of repetition you can use `are`. E.g. something like this: 
```clojure
(deftest combine-adjacent-strings-tests
  (are [x y] (= x (parse-helper/combine-adjacent-strings y))
     [] []
     ["some text"] ["some" " " "text"]
     ["some text" [:link] "around a link"] ["some" " " "text" [:link] "around " "a link"]
     [{:something nil} "more text" [:link] "between elements" 39]
      [{:something nil} "more" " " "text" [:link] "between" " " "elements" 39]
     [{:a 1 :b 2} 3 ["leave" "intact"]]
      [{:a 1 :b 2} 3 ["leave" "intact"]]))
```

Reply:
Done. That does make it a bit easier to compare the before and after values. I kept all inputs/outputs on their own line to make it easy to see where one ends and the other begins.